ha/fault_tolerance/rest_api.py

```diff
@@ -0,0 +1,256 @@
+import signal
+import ssl
+import time
+from abc import ABC
+from aiohttp import web
+import asyncio
+from asyncio import CancelledError as AsyncioCancelledError
+from concurrent.futures import CancelledError as ConcurrentCancelledError
+
+from cortx.utils.log import Log
+from cortx.utils.conf_store import Conf
+from ha import const
+
+class Response:
+    pass
+
+class CcRestApi(ABC):
```

Comment:
Why derived from ABC?
Is it a abstract class?

Reply:
this calls to be abstract as we will be handling a single server we don't want multiple instances of the server.
so all methods are static or class methods.

Reply:
Ok.